src/main/java/ratpack/zipkin/internal/MethodCapturingRequestSpec.java

```diff
@@ -60,7 +67,8 @@ public RequestSpec headers(Action<? super MutableHeaders> action) throws Excepti
 
   @Override
   public RequestSpec method(HttpMethod method) {
-    capturedMethod = method;
+    this.capturedMethod = method;
+    span.set(impl.handler.handleSend(impl.injector, getHeaders(), this));
```

Comment:
I spent some time today trying to get the ITHttpClient tests to pass.  In the end I was able to get all tests to pass except requests with nested spans.  I even tried switching the CurrentTraceContext to the RatpackCurrentTraceContext implementation.  This also failed but in different ways.  The end result is that the current span was never found in the CurrentTraceContext so the span parentId would always be null.

Ratpack is very peculiar about what thread things are accessed on. 

At any rate, I will keep plugging along with this and try to figure out a solution.